docs/resources/control-flow/backend-logic/backend-query/query-collection.md

```diff
@@ -113,4 +149,21 @@ You can follow similar steps for using the record data on the other widgets as w
         allowFullScreen
         allow="clipboard-write">
     </iframe>
-</div>
\ No newline at end of file
+</div>
+<p></p>
+</TabItem>
+</Tabs>
+
+## FAQs
+<details>
+<summary>Why are real-time updates for Supabase not working?</summary>
+<p>
+First, ensure that the **Single Time Query** option is disabled in the query where you've added it. Then, verify that the real-time feature is enabled for your table in Supabase project. You can find this option in the top-right corner of the table viewer.
```

Comment:
"for your table in your Supabase project"

Reply:
ok, corrected question text as 'Why aren't real-time updates working for my table in Supabase project?'